### ISSUE TITLE
`cadwork.projection_type` docstring implementation inconsistency

### ISSUE THREAD

**Issue by chenkasirer** [cwapi3d/cwapi3dpython#188]
According to the docs the enum has following values:

from https://docs.cadwork.com/projects/cwapi3dpython/en/latest/documentation/camera_data/?h=projection#src.cadwork.projection_type.projection_type
```
@unique
class projection_type(IntEnum):
    """projection type

    Examples:
        >>> cadwork.projection_type.Perspective
        Perspective
    """
    Perspective = 1
    """"""
    Orthographic = 2
    """"""

    def __int__(self) -> int:
        return self.value
```

However, implementation seems to be zero-based:
```
Python 3.10.10 (main, Mar 31 2023, 08:23:52) [MSC v.1935 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
import cadwork
cadwork.projection_type(1)
<projection_type.Orthographic: 1>
cadwork.projection_type(0)
<projection_type.Perspective: 0>
cadwork.projection_type(2)
<projection_type.???: 2>
cadwork.projection_type
<class 'cadwork.projection_type'>
cadwork.projection_type.Orthographic
<projection_type.Orthographic: 1>
cadwork.projection_type.Perspective
<projection_type.Perspective: 0>
```

cadwork build: `493`
cwapi3d version: `30.498.0`

**Comment by Brunner246** [comment]
fixed in version `version = "30.0.533"`

**Comment by Brunner246** [comment]
@chenkasirer 

https://github.com/cwapi3d/cwapi3dpython/blob/33767f918955577735f24c31ead396da58712bf5/src/cadwork/projection_type.pyi#L12
